internal/client_db_connector/connector.go

```diff
@@ -112,14 +113,14 @@ func (d *ClientDbConnectorImpl) ImportFromS3(ctx context.Context, clientDb types
 	return response.GetOperation().GetId(), nil
 }
 
-func (d *ClientDbConnectorImpl) GetOperationStatus(ctx context.Context, clientDb types.YdbConnectionParams, operationId string) (types.YdbOperationInfo, error) {
+func (d *ClientDbConnectorImpl) GetOperationStatus(ctx context.Context, clientDb types.YdbConnectionParams, operationId string) (*Ydb_Operations.GetOperationResponse, error) {
 	clientDbConnectionString := clientDb.Endpoint + clientDb.DatabaseName
```

Comment:
We could move it to a separate function with a lot of debug logs (if users specifies this incorrectly)